tests/foreman/ui/test_gpgkey.py

```diff
@@ -213,22 +200,24 @@ def test_negative_create_6(self, name):
 
         """
         with Session(self.browser) as session:
-            make_gpgkey(
-                session,
-                key_content=self.key_content,
-                name=name,
-                org=self.organization.name,
-            )
-            self.assertIsNotNone(
-                self.gpgkey.wait_until_element(common_locators['alert.error'])
-            )
-            self.assertIsNone(self.gpgkey.search(name))
+            for name in invalid_names_list():
```

Comment:
Nope I meant to use invalid_names_list.  I have no idea why we were testing blank earlier.  Since the UI save button will not be enabled with blank or space values :)